simulator/simulator.py

```diff
@@ -78,32 +79,51 @@
 board.register_device(args.device)
 
 if args.sd:
+    # pylint: disable=unused-import
```

Comment:
I think we can customize pylint so we don't have to pollute the code with all these comments

Reply:
Agreed